dashboard/src/components/TicketCard.vue

```diff
@@ -61,11 +61,28 @@
 				<img
 					:src="ticket.qr_code"
 					:alt="__('QR Code')"
-					class="w-20 h-20 contrast-100 brightness-100"
+					:title="__('Click to enlarge')"
+					class="w-20 h-20 contrast-100 brightness-100 cursor-pointer hover:opacity-80 transition-opacity"
+					@click.stop="showQRExpanded = true"
 				/>
+				<p
+					class="text-xs text-ink-gray-5 mt-1 cursor-pointer hover:text-ink-gray-7"
+					@click.stop="showQRExpanded = true"
+				>
+					{{ __("Tap to enlarge") }}
+				</p>
 			</div>
 		</div>
 
+		<!-- QR Code Expanded Dialog -->
+		<Dialog v-model="showQRExpanded" :options="{ title: __('QR Code'), size: 'lg' }">
+			<template #body-content>
+				<div class="flex justify-center">
+					<img :src="ticket.qr_code" :alt="__('QR Code')" class="w-full max-w-sm" />
+				</div>
+			</template>
+		</Dialog>
```

Comment:
Let's create a component for this, since it has been used in two places?

BTW `v-model` should work even when the component gets created.